### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by limavfabio**
This PR setups the initial setup of a react app with create-react-app and Microverse provided linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,42 @@
-# Getting Started with Create React App
+# Project Name
```
Comment (DammyShittu) [inline README.md]: - [x] Kindly give your project a proper title

File: README.md [hunk 2]
```diff
@@ -1,70 +1,42 @@
...
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- Clone this repo
+- Install dependencies 
+```
+npm install
+```
+- Run build script
+```
```
Comment (DammyShittu) [inline README.md:23]: - [x] After installing dependencies we need to start the app with `npm start`. Kindly add this command.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @limavfabio,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by dasileker — APPROVED** [review]
### **Hi @limavfabio,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**